lib/impala.rb

```diff
@@ -16,10 +16,10 @@
 module Impala
   DEFAULT_HOST = 'localhost'
   DEFAULT_PORT = 21000
-  class Error < StandardError; end
```

Comment:
Is there a reason to remove this exception class and make all subclasses inherit from StandardError?